src/ASM/ASMbase.C

```diff
@@ -21,6 +21,10 @@
 #include "Utilities.h"
 #include <algorithm>
 
+#ifdef USE_OPENMP
+#include <omp.h>
+#endif
+
```

Comment:
Kan hende. Trodde kanskje du hadde en egen assembly blokk for disse globale LM-en der det kunne håndteres. må evt. se det som falt bort før jeg kan men noe mer.